filesystem/iso9660.ksy

```diff
@@ -214,60 +348,201 @@ types:
+        type: susp_unknown
+        if: signature == su_signature::rras_amiga_specific
+      - id: susp_ce # CE
+        type: susp_unknown
+        if: signature == su_signature::susp_continuation_area
+      - id: rrip_cl # CL
+        type: susp_unknown
+        if: signature == su_signature::rrip_child_link
+      - id: susp_er # ER
+        type: susp_unknown
+        if: signature == su_signature::susp_extensions_reference
+      - id: susp_es # ES
+        type: susp_unknown
+        if: signature == su_signature::susp_extension_selector
+      - id: rrip_nm # NM
+        type: susp_unknown
+        if: signature == su_signature::rrip_alternate_name
+      - id: susp_pd # PD
+        type: susp_unknown
+        if: signature == su_signature::susp_padding_field
+      - id: rrip_pl # PL
+        type: susp_unknown
+        if: signature == su_signature::rrip_parent_link
+      - id: rrip_pn # PN
+        type: susp_unknown
+        if: signature == su_signature::rrip_posix_device_number
+      - id: rrip_px # PX
+        type: susp_unknown
+        if: signature == su_signature::rrip_posix_file_attributes
+      - id: rrip_re # RE
+        type: susp_unknown
+        if: signature == su_signature::rrip_relocated_directory
+      - id: rrip_rr # RR
+        type: susp_unknown
+        if: signature == su_signature::rrip_extensions_in_use_indicator
+      - id: rrip_sf # SF
+        type: susp_unknown
+        if: signature == su_signature::rrip_sparse_file
+      - id: rrip_sl # SL
+        type: susp_unknown
+        if: signature == su_signature::rrip_symbolic_link
+      - id: susp_sp # SP
+        type: susp_unknown
+        if: signature == su_signature::susp_indicator
+      - id: susp_st # ST
+        type: susp_unknown
+        if: signature == su_signature::susp_terminator
+      - id: rrip_tf # TF
+        type: susp_unknown
+        if: signature == su_signature::rrip_time_file
+      - id: rrzf_zf # ZF
+        type: susp_unknown
+        if: signature == su_signature::rrzf_zisofs
+  su_headers:
+    seq:
+      - id: header
+        type: su_header
+        repeat: eos
+  directory_records:
+    doc: |
+      First item "." it points to it self
+      Second item ".." it points to the parent, or also to self if it the root
+    seq:
+      - id: directory_record
+        type: directory_record
+        repeat: until
+        repeat-until: _.len_dr == 0
+  directory_record:
+    doc-ref: ecma-119 9.1
+    seq:
+      - id: len_dr
+        doc-ref: ecma-119 9.1.1
+        type: u1
+      - id: ext_attr_rec_len
+        doc-ref: ecma-119 9.1.2
+        type: u1
+        if: len_dr > 0x0
+      - id: location_of_extent
+        doc-ref: ecma-119 9.1.3
+        type: u4bi
+        if: len_dr > 0x0
+      - id: data_len
+        doc-ref: ecma-119 9.1.4
+        type: u4bi
+        if: len_dr > 0x0
+      - id: rec_date_time
+        doc-ref: ecma-119 9.1.5
+        type: recdatetime
+        if: len_dr > 0x0
```

Comment:
I gess we need to move the comparison into an instance and use in if that instance.